apis/v1/binding_types.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CRPTrackingLabel is the label that points to the cluster resource policy that creates a resource binding.
+const CRPTrackingLabel = labelPrefix + "parentCRP"
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,categories={fleet},shortName=rb
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="ResourceBindingApplied")].status`,name="Applied",type=string
+// +kubebuilder:printcolumn:JSONPath=`.metadata.creationTimestamp`,name="Age",type=date
+
+// ClusterResourceBinding is represents a scheduling decision that binds a group of resources to a cluster.
+// it must have CRPTrackingLabel that points to the cluster resource policy that creates it.
+type ClusterResourceBinding struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// The desired state of ClusterResourceBinding.
+	// +required
+	Spec ResourceBindingSpec `json:"spec"`
+
+	// The observed status of ClusterResourceBinding.
+	// +optional
+	Status ResourceBindingStatus `json:"status,omitempty"`
+}
+
+// ResourceBindingSpec defines the desired state of ClusterResourceBinding.
+type ResourceBindingSpec struct {
+	// ResourcePolicyName is the name of the resource policy that this resource binding points to.
+	ResourcePolicyName string `json:"resourcePolicyName"`
+
+	// ResourceSnapshotIndex is the index of the resource snapshot to which
+	// that this resource binding points to.
+	ResourceSnapshotIndex int32 `json:"resourceSnapshotIndex"`
```

Comment:
```suggestion
	// ResourceSnapshotName is the name of the resource snapshot that this resource binding points to. If the resources are divided into multiple snapshots because of the resource size limit, it will point to the name of the parent snapshot.
	ResourceSnapshotName string `json:"resourceSnapshotName"`
```
you may miss this comment? https://github.com/Azure/fleet/pull/367#discussion_r1206637501